.ccache/ccache.conf

```diff
@@ -0,0 +1,32 @@
+# For complete documentation, see https://ccache.dev/manual/4.6.1.html#_configuration
+
+# This option specifies the maximum size of the cache. Use 0 for no limit. The default value is 5G.
+# Available suffixes: k, M, G, T (decimal) and Ki, Mi, Gi, Ti (binary). The default suffix is G
+max_size = 10G
+
+# This option specifies where ccache will keep its cached compiler outputs.
+cache_dir = /mnt/ccache/
+
+#If true, ccache will compress data it puts in the cache. However, this option has no effect on how files are
+#retrieved from the cache; compressed and uncompressed results will still be usable regardless of this option.
+# The default is true.
+compression = true
+
+# This option determines the level at which ccache will compress object files using the real-time compression
+# algorithm Zstandard. It only has an effect if compression is enabled (which it is by default). In restricted space environments,
+# it is recommended to set the value of this parameter to 19 (or via environment parameter CCACHE_COMPRESSLEVEL)
+compression_level = 1
```

Comment:
why compress if you use the lowest possible compression rate?

Reply:
- Lowest compression rate still compress it more than 80% and the space on local build is not unlimited, so I wanted to have compression still.
- I first had it on 19 (higest for all versions) to check how it will be on CI. The differnce in speed in 5-10% 19 vs 1.
- I prefer speed over space for local, and space over speed on CI. still, 1 gives around 80% compression on very minimal cost 1-3% of the speed.